snap/info.go

```diff
@@ -163,6 +162,8 @@ type Info struct {
 	Prices  map[string]float64 `yaml:"prices,omitempty" json:"prices,omitempty"`
 	MustBuy bool
 
+	Contact string
```

Comment:
if it comes from snap.yaml it needs to go higher in this struct, there's a comment that says thing below come from the store only